src/commonControls/listControl/ListControlAPL.ts

```diff
@@ -25,19 +27,21 @@ export namespace ListControlAPLPropsBuiltIns {
      * For information about the TextListTemplate, see following doc:
      * https://developer.amazon.com/en-US/docs/alexa/alexa-presentation-language/apl-alexa-text-list-layout.html
      */
-    export const TextList: DeepRequired<ListControlAPLProps> = {
-        enabled: true,
-        requestValue: {
-            document: TextListDocumentGenerator(),
-            dataSource: TextListDataSourceGenerator((choiceId) => choiceId),
-            customHandlingFuncs: [],
-        },
-        requestChangedValue: {
-            document: TextListDocumentGenerator(),
-            dataSource: TextListDataSourceGenerator((choiceId) => choiceId),
-            customHandlingFuncs: [],
-        },
-    };
+    export function TextList(valueRenderer: (value: string, input: ControlInput) => string) {
+        return {
+            enabled: true,
+            requestValue: {
+                document: TextListDocumentGenerator(),
```

Comment:
nit: document/dataSource function names should start lowercase

Reply:
Fixed